docs/_spec/03-types.md

```diff
@@ -211,6 +206,51 @@ G[S, Int]             // illegal: S constrains its parameter to
                       //   that conforms to Int
 ```
 
+#### Wildcard Type Argument
+
+<!-- TODO Update the syntax to use '?' -->
+```ebnf
+WildcardType   ::=  ‘_’ TypeBounds
+```
+
+A _wildcard type argument_ is of the form `_´\;´>:´\,L\,´<:´\,U´`.
+Both bound clauses may be omitted.
+If a lower bound clause `>:´\,L´` is missing, `>:´\,´scala.Nothing` is assumed.
+If an upper bound clause `<:´\,U´` is missing, `<:´\,´scala.Any` is assumed.
+A wildcard type is a shorthand for an existentially quantified type variable, where the existential quantification is implicit.
+
+A wildcard type must appear as a type argument of a parameterized type.
+The parameterized type to which the wildcard type is applied cannot be an abstract type constructor.
+The wildcard type argument is kind-polymorphic.
```

Comment:
It's not really kind-polymorphic. Its bounds are inferred (when not explicitly written) to conform to the expected kind and type bounds of its type constructor.